controller/kubernetes_configmap_controller.go

```diff
@@ -73,6 +73,17 @@ func NewKubernetesConfigMapController(
 		UpdateFunc: func(old, cur interface{}) { kc.enqueueConfigMapChange(cur) },
 		DeleteFunc: kc.enqueueConfigMapChange,
 	})
+
+	ds.StorageClassInformer.AddEventHandler(
+		cache.FilteringResourceEventHandler{
+			FilterFunc: isLonghornStorageClass,
+			Handler: cache.ResourceEventHandlerFuncs{
+				UpdateFunc: func(old, cur interface{}) { kc.enqueueConfigMapForStorageClassChange(cur) },
+				DeleteFunc: kc.enqueueConfigMapForStorageClassChange,
+			},
+		},
+	)
+
 	kc.cacheSyncs = append(kc.cacheSyncs, ds.ConfigMapInformer.HasSynced)
```

Comment:
@shuo-wu We are missing the wait for cache sync for the storage class informer.

Reply:
Sorry for missing it! 🤣